modules/driver_dw1000/dw1000.h

```diff
@@ -78,6 +112,9 @@ struct dw1000_instance_s {
     struct spi_device_s spi_dev;
     uint32_t reset_line;
     struct dw1000_config_s config;
+    struct pubsub_topic_s *irq_topic;
+    struct pubsub_listener_s irq_listener;
+    struct worker_thread_listener_task_s irq_listener_task;
```

Comment:
Are you going to get rid of these? I didn't want driver_dw1000 to have a dependency on pubsub or worker thread.